server/service/src/sync/sync_on_central/mod.rs

```diff
@@ -113,9 +67,62 @@ pub async fn pull(
     .map(SyncRecordV6::from)
     .collect();
 
+    warn!("Sending records as central server: {:#?}", records);
```

Comment:
This is very useful for testing, but should remove/change to debug! before merging...